cdc/processor/processor.go

```diff
@@ -486,6 +488,18 @@ func isProcessorIgnorableError(err error) bool {
 	return false
 }
 
+// needPullerSafeModeAtStart returns true if the scheme is mysql compatible.
+// pullerSafeMode means to split all update kv entries whose commitTS
+// is older then the start time of this changefeed.
+func needPullerSafeModeAtStart(sinkURIStr string) (bool, error) {
+	sinkURI, err := url.Parse(sinkURIStr)
+	if err != nil {
+		return false, cerror.WrapError(cerror.ErrSinkURIInvalid, err)
+	}
+	scheme := sink.GetScheme(sinkURI)
+	return sink.IsMySQLCompatibleScheme(scheme), nil
```

Comment:
Other sinks have similar problems, so it's best to have consistent behavior.

Reply:
1. Other sinks can't detect error like "duplicate entry", so I think support them here can't prevent the problem, and even doesn't alleviate the problem. So we should find a more systematic way for other sinks if needed;
2. The action to split all update events to delete and insert events at restart time is visible to other sinks, so users of these kinds of sinks may be confused for this kind of changes;

Reply:
I suggest we open an new issue to track the overall solution, including for other sink types. And this PR focus on resolving the mysql issue.